torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1012,29 +1012,31 @@ void ProcessGroupNCCL::waitForDumpOrTimeout(
   TORCH_CHECK(fut.valid(), "Expected a valid future");
 
   auto futStatus = fut.wait_for(std::chrono::seconds(timeout_sec));
```

Comment:
I am not sure the default times are set properly, e.g., the default timeout_sec is 30s, so we wait for a max of 30s for local dump here, which should long pass the wakeUpTime point (default 2 secs after init dumping)?

Reply:
yea, we need to clean up the time variables.